frontend/src/main/components/Utils/UsersTableUtils.js

```diff
@@ -0,0 +1,31 @@
+import { toast } from "react-toastify";
+
+export function cellToAxiosParamsToggleAdmin(cell){
+    return{
+        url: "/api/admin/users/toggleAdmin",
+        method: "POST",
+        params:{
+            id: cell.row.values.id
```

Comment:
Compare this line of code from the backend implementation of this route:


```java
    public Object toggleAdmin( @Parameter(name = "githubId", description = "Integer, github id of user to toggle their admin status", example = "1", required = true) @RequestParam Integer githubId){
```

Do you see what's wrong?